src/actions/population.js

```diff
@@ -15,6 +18,14 @@ export function fetchPopulations() {
   };
 }
 
+export function fetchPopulation(id) {
```

Comment:
This action won't work as the API isn't working yet. Better to pass through the dummy data here in the payload instead of in the container so you can test the actions correctly.